styleguide.config.js

```diff
@@ -10,7 +10,38 @@ module.exports = {
   assetsDir: 'public',
   styleguideDir: 'docs',
   title: `Exchange Solutions React Components v${version}`,
-  template: 'config/template.html',
+  template: {
+    meta: [
+      { charset: "utf-8" },
+      { 
+        name: "viewport", 
+        content: "width=device-width, initial-scale=1.0"
+      }
+    ],
+    head: {
+      raw: `<style>
+        @font-face {
+        font-family: 'indv-mkt-icons';
+        src: url('webfonts/indv-mkt-icons.eot?3tk8wr');
+        src: url('webfonts/indv-mkt-icons.eot?3tk8wr#iefix') format('embedded-opentype'),
+          url('webfonts/indv-mkt-icons.ttf?3tk8wr') format('truetype'),
+          url('webfonts/indv-mkt-icons.woff?3tk8wr') format('woff'),
+          url('webfonts/indv-mkt-icons.svg?3tk8wr#indv-mkt-icons') format('svg');
+        font-weight: normal;
+        font-style: normal;
+        }
+
+        body {
+          font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+        }
+      </style>
+      <style>#forkongithub a{background:#000;color:#fff;text-decoration:none;font-family:arial,sans-serif;text-align:center;font-weight:bold;padding:5px 40px;font-size:1rem;line-height:2rem;position:relative;transition:0.5s;}#forkongithub a:hover{background:#c11;color:#fff;}#forkongithub a::before,#forkongithub a::after{content:"";width:100%;display:block;position:absolute;top:1px;left:0;height:1px;background:#fff;}#forkongithub a::after{bottom:1px;top:auto;}@media screen and (min-width:800px){#forkongithub{position:fixed;display:block;top:0;right:0;width:200px;overflow:hidden;height:200px;z-index:9999;}#forkongithub a{width:200px;position:absolute;top:60px;right:-60px;transform:rotate(45deg);-webkit-transform:rotate(45deg);-ms-transform:rotate(45deg);-moz-transform:rotate(45deg);-o-transform:rotate(45deg);box-shadow:4px 4px 10px rgba(0,0,0,0.8);}}</style>
+      `
+    },
+    body: {
+      raw: '<span id="forkongithub"><a href="https://github.com/WTW-IM/es-components">Fork me on GitHub</a></span>'
```

Comment:
I would just remove my updates from #121 and replace it with [these changes](https://github.com/WTW-IM/es-components/pull/134/commits/244d325233aa65badb4b81bae6f1ceaf9edaf124).

Reply:
Done and done.